custom-words.txt

```diff
@@ -319,6 +319,7 @@ BRCPF
 browsable
 bruceper
 Bsaling
+buildout
```

Comment:
nit.. is buildout a real word?

Reply:
`buildout` here refers to the status of Azure Cosmos DB in a region. For new regions which are still in buildout the status can be `Initializing` based on the stage of regional buildout.